docs/config/deployer.md

```diff
@@ -485,6 +485,15 @@ This config is used for tests that require an existing database dump that contai
 
 The path to a file containing a list of credentials for the controllers to use. If present, it is used to automatically upload it to the agents and override the agent's config's own [`UsersFilePath`](config.md/#UsersFilePath).
 
+## EnableNetPeekMetrics
+
+*bool*
+
+If true, enables the collection of fine grained networking metrics through the [netpeek](https://github.com/streamer45/netpeek) utility.
+
+> [!WARNING]
+> These metrics can introduce a not negligible computational overhead on high traffic deployments.
```

Comment:
Is this something we should measure at some point? Specially if we want to run it in ceiling tests

Reply:
@agarciamontoro I [measured a bit](https://snapshots.raintank.io/dashboard/snapshot/0TA9RUvi6wyzUaug9H8BdcH6N4o6c6mm?orgId=0&from=1726796371225&to=1726797271225) while testing this PR. The binary also exposes go-process metrics so it's easy to track CPU time if needed.